test/repositories_include_test.rb

```diff
@@ -74,6 +74,24 @@ def main
       expect(ret).to eq(:again)
     end
 
+    context "if the URL cannot be expanded (is invalid)" do
+      before do
+        # URL expansion returns nil (bsc#1059744)
+        allow(Yast::Pkg).to receive(:ExpandedUrl).with(url).and_return(nil)
+      end
+
+      it "returns :again symbol" do
+        puts url
```

Comment:
Is this `puts url` intended?

Reply:
Ah, it was for debugging... I'll remove it.